packages/core/security/core-security-server-internal/src/fips/fips.test.ts

```diff
@@ -76,11 +76,11 @@ describe('fips', () => {
         return 1;
       });
 
-      config = { experimental: { fipsMode: { enabled: false } } };
+      securityConfig = { experimental: { fipsMode: { enabled: false } } };
       const logger = loggingSystemMock.create().get();
 
       try {
-        checkFipsConfig(config, logger);
+        checkFipsConfig(securityConfig, {}, {}, logger);
```

Comment:
nit: what do you think about adding this on the next line?

```ts
fail('FIPS config check did not throw')
```

(ditto for other try catches like this)

Reply:
++ added in latest commit